tox.ini

```diff
@@ -29,3 +29,8 @@ commands =
 [testenv:integration{,-py39,-py310,-py311}]
 description = Run integration tests
 commands = pytest {posargs:tests/integration}
+
+[flake8]
+# W503 - Line break occurred before a binary operator
+ignore=W503
+max-line-length=160
```

Comment:
Is there something related to using pyproject.tomal that requires this change to flake8?

Reply:
flake8 is not supported in pyproject.toml directly. We need to install third-party libraries to make it work. So add changes to `tox.ini`